src/coreclr/System.Private.CoreLib/src/System/Reflection/RuntimeCustomAttributeData.cs

```diff
@@ -1879,7 +1879,7 @@ private static object CreateCustomAttributeInstance(RuntimeModule module, Runtim
         {
             if (module is null)
             {
-                throw new ArgumentNullException(SR.Arg_InvalidHandle);
+                throw new ArgumentNullException(null, SR.Arg_InvalidHandle);
             }
```

Comment:
I'm curious why you didn't collapse the `if (module is null)` and use `ArgumentNullException.ThrowIfNull(module, SR.Arg_InvalidHandle);`.

Reply:
The second argument to ThrowIfNull is the parameter name, not message. And the access to SR.Arg_InvalidHandle would happen even in the expected case where module is not null.

Reply:
Ah, I didn't catch that `SR.Arg_InvalidHandle` is a message (as opposed to the argument name). Thanks.